src/test/java/repsmax/StorageTest.java

```diff
@@ -0,0 +1,47 @@
+package repsmax;
+
+import org.junit.jupiter.api.Test;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Scanner;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class StorageTest {
+
+    @Test
+    public void testSave_andLoad_tasksArePersisted() throws Exception {
```

Comment:
Perhaps a clearer description of test inputs can be used for method name (unitBeingTested_descriptionOfTestInputs_expectedOutcome).